docs/user-guide/knowledge-distillation.rst

```diff
@@ -0,0 +1,362 @@
+.. include:: /content/nemo.rsts
+
+SFT with Knowledge Distillation
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+Knowledge distillation is a technique in which a smaller (student) model learns from a larger (teacher) model. The goal is to "distill" information from the teacher to the student.
+Compared to standard SFT which trains the model to predict the next token, knowledge distillation allows more calibrated information passing from the teacher to the student.
+There are two primary benefits of knowledge distillation compared to standard supervised fine-tuning: (1) convergence in fewer training tokens, and (2) improved accuracy.
+
+There are many variants of knowledge distillation. NeMo Aligner supports training the student model to match the top-K logits of the teacher model. In this tutorial, we will go through fine-tuning a 2B student using a fine-tuned Nemotron 8B chat model.
+
+Step 1: Obtain the fine-tuned teacher and pre-trained student models
+####################################################################
+To start, we must first download both the pre-trained student and fine-tuned teacher models
+
+.. tab-set::
+
+    .. tab-item:: 2B GPT Student
+        :sync: key1
+
+        #. Get the 2B checkpoint: ``wget https://huggingface.co/nvidia/GPT-2B-001/resolve/main/GPT-2B-001_bf16_tp1.nemo``
+        #. Extract the NeMo File to a folder with ``mkdir student_checkpoint && tar -xvf GPT-2B-001_bf16_tp1.nemo -C student_checkpoint``
+        #. And then run the script to convert from old NeMo checkpoint to Megatron-Core checkpoint. The script is located `here <https://github.com/NVIDIA/NeMo/blob/66646b83737d9a0facb1d8d714e0424fc86ec21a/scripts/checkpoint_converters/convert_gpt_nemo_to_mcore.py>`__.
+            
+            .. code-block:: bash 
+
+               python convert_gpt_nemo_to_mcore.py \
+                  --input_name_or_path ./student_checkpoint \
+                  --output_path ./2b_student.nemo
+
+    .. tab-item:: Nemotron 8B Teacher
+        :sync: key2
+
+        #. Download the `Llama3-8B LLM model and tokenizer <https://huggingface.co/meta-llama/Meta-Llama-3-8B>`__ into the model's folder. You can use the HuggingFace CLI for this:
```

Comment:
fix spacing (HF is two words)

        #. Download the `Llama3-8B LLM model and tokenizer <https://huggingface.co/meta-llama/Meta-Llama-3-8B>`__ into the model's folder. You can use the Hugging Face CLI for this: